docs/develop/basics.md

```diff
@@ -0,0 +1,117 @@
+
+You can find the pre-built configurations in our `Configurations` namespace in most SDKs. Here are some of the pre-built configurations that
+you might be interested in.
+
+### Laptop
+
+`Configurations.Laptop` is a development environment, just for poking around on your... laptop :) It has relaxed timeouts and assumes that your network latencies might be a bit high.
+
+<SdkExampleTabs snippetId={'API_ConfigurationLaptop'} />
+
+### Browser
+
+`Configurations.Browser` is a special configuration available in the [web SDK](/develop/sdks/web). It has relaxed timeout settings since latencies can be extremely variable in individual users' browsers, depending on their network.
+
+### InRegion - Default
+
+`Configurations.InRegion.Default` is the recommended configuration for most server-side use cases, where you are making requests to Momento a service hosted in the same AWS or GCP region. It has more aggressive timeouts and retry behavior than the Laptop config, so it will fail faster and allow your application to fall back to your database or other data source more quickly. This helps ensure that your application doesn't bottleneck on Momento during a network or service outage.
+
+<SdkExampleTabs snippetId={'API_ConfigurationInRegionDefault'} />
+
+### InRegion - Low Latency
+
+`Configurations.InRegion.LowLatency` is a configuration that prioritizes keeping p99.9 latencies as low as possible, potentially sacrificing some throughput to achieve this. It will time out more quickly than the Default configuration. Use this configuration if the most important factor is to ensure that cache unavailability doesn't force unacceptably high latencies for your own application and you want to fall back to application logic more quickly if the cache hasn't responded.
+
+<SdkExampleTabs snippetId={'API_ConfigurationInRegionLowLatency'} />
+
+### Lambda
+
+`Configurations.Lambda` is a configuration that is available in some SDKs, and is optimized for the AWS Lambda environment. It has some configuration settings designed to pre-warm the client on Lambda cold starts, and to ensure the connection is re-established proactively if a Lambda remains idle for long enough for the connection to time out.
+
+<SdkExampleTabs snippetId={'API_ConfigurationILambda'} />
+
+If you do need to customize your configuration beyond what our pre-builts provide, you can build your own `Configuration`
+object.  See the examples in the `Configurations` namespace in the source code of your SDK  to see how they are constructed.
+
+We hope that these configurations will meet the needs of most users, but if you find them lacking in any way, please
+open a github issue, or contact us at `support@momentohq.com`. We would love to hear about your use case so that we
```

Comment:
```suggestion
open a GitHub issue on that SDK repo, or contact us at `support@momentohq.com`. We would love to hear about your use case so that we
```